src/home5task1test.py

```diff
@@ -0,0 +1,98 @@
+# 3 --------------------------------------------------------------------------
```

Comment:
in fact it tests only one file and only it's fixed func names. too narrow test cases